### DISCUSSION TITLE
Python Updates?

### DISCUSSION THREAD

**Discussion by dredgy** [maxileith/homebridge-appletv-enhanced#537] [Q&A] [answered]
Hey would like to use this plugin, wondering if there were any plans to update the Python support? 

`Python 3.12.3 is installed. However, only Python 3.8 to 3.11 is supported.`

**Comment by maxileith** [comment]
See #433.

**Comment by maxileith** [comment, marked as answer]
Python 3.12 is supported since 1.6.0
